code/data_structures/src/tree/binary_tree/avl_tree/avl__tree.py

```diff
@@ -0,0 +1,262 @@
+#import random, math
+
+outputdebug = False 
+
+def debug(msg):
+    if outputdebug:
+        print msg
+
+class Node():
+    def __init__(self, key):
+        self.key = key
+        self.left = None 
+        self.right = None 
+
+
```

Comment:
Remove extra newlines according to PEP 8.

Reply:
Removed